qiskit/visualization/matplotlib.py

```diff
@@ -520,7 +520,7 @@ def _draw_ops(self, verbose=False):
                 this_anc = q_anchors[q_idxs[0]].gate_anchor
 
                 if verbose:
-                    print(op)
+                    print(op.name)
```

Comment:
This shouldn't be changed in a commit for latex

Reply:
Removed